src/utils/i18n.ts

```diff
@@ -206,19 +206,19 @@ export function isPhrases(phrases: Phrases[string]): phrases is Phrases {
 }
 
 /**
- * Generates viable subsets from a given locale.
- *
+ * Creates a fallback chain for a locale.
+ * 
  * Examples:
  * ```
- * new LanguageTag('zh-Hant-HK').makeSubsets()    // ['zh-Hant', 'zh']
- * new LanguageTag('zh-HK').makeSubsets()         // ['zh']
- * new LanguageTag('zh-Hant').makeSubsets()       // ['zh']
- * new LanguageTag('zh').makeSubsets()            // []
+ * getFallbackPattern('zh-Hant-HK')    // ['zh-Hant-HK', 'zh-Hant', 'zh']
+ * getFallbackPattern('zh-HK')         // ['zh-HK', 'zh']
+ * getFallbackPattern('zh-Hant')       // ['zh-Hant', 'zh']
+ * getFallbackPattern('zh')            // ['zh']
  * ```
  */
-function makeSubsets(locale: string): string[] {
-  const subsets: string[] = [];
+export function getFallbackPattern(locale: string): string[] {
   const languageTag = new LanguageTag(locale);
+  const subsets: string[] = [languageTag.toString()];
```

Comment:
I changed it to be inclusive.

Reply:
Nitpick: to be consistent with the new function name, maybe `subsets` should be renamed to `fallbacks` or something?